oap-server/server-receiver-plugin/skywalking-trace-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/trace/provider/TraceServiceModuleConfig.java

```diff
@@ -54,6 +54,9 @@
     @Setter
     @Getter
     private UninstrumentedGatewaysConfig uninstrumentedGatewaysConfig;
+    @Setter
+    @Getter
+    private NoUpstreamRealAddressAgentConfig noUpstreamRealAddressAgentConfig;
```

Comment:
We support list here, take a look at downsampling config. And you should put LUA as the default element.

Reply:
done